docs/docs/50-running/55-memory.md

```diff
@@ -0,0 +1,103 @@
+:::
+
+## Run
+Setting `memory` via `acorn run` has the highest order of precedence. When setting this, you operate under a new flag `--memory` (`-m` for short).
+
+### --memory
+When setting this value, you have two options - globally or per workload and you can do a combination of both. When setting the memory globally for that Acorn, you just define the memory you would like to set.
+
+```shell
+acorn run -m 512Mi foo
+```
+
+This will set all workloads in the `foo` acorn to have `512Mi` of memory. Adjacently, you can set the memory of each individual workload in the Acorn using a `workload=memory` pattern. 
+
+```shell
+acorn run -m nginx=512Mi foo
+```
+
+This will only update Acorn's `nginx` workload to have `512Mi` of memory.
+
+Finally, you can do a combination of both.
+
+```shell
+acorn run -m 256Mi,nginx=512Mi foo
+```
+
+This sets all workloads in the `foo` acorn to have `256Mi` of memory except for the `nginx` workload which will have `512Mi` of memory.
+
+
+## Acornfile
+Setting the memory via the Acornfile uses the `memory` property that is attached to all `workloads` (`containers` and `jobs`).
+
+```acorn
+containers: {
+    nginx: {
+        image: "nginx"
+        ports: publish: "80/http"
+        files: {
+            "/usr/share/nginx/html/index.html": "<h1>My first Acorn!</h1>"
+        }
+        memory: 512Mi
+    }
+}
+```
+
+> **Note**: This can be overwritten by `acorn run --memory`.
+
+## Install
+There are two `install` flags for setting memory: `--workload-memory-default` and `--workload-memory-maximum`. Their values can both be viewed by running `acorn info`.
+
+### --workload-memory-default
+This flag is responsible for setting the memory amount that will get defaulted to should no other value be found.
+
+```shell 
+acorn install --workload-memory-default 512Mi
+```
+
+Running the above will set all Acorns on the cluster (current and future) to use `512Mi` as their default memory.
+
+### --workload-memory-maximum
+This flag sets a maximum that when exceeded prevents the offending Acorn from being installed.
+
+```shell
+acorn install --workload-memory-maximum 1Gi`
+```
+
+This will set it so all Acorns on this cluster will be unable to install should they exceed `1Gi` of memory.
+
+## No limit
+It is possible to set any of these flags to have no limit on memory by simply setting their value to `0`. However, running an Acorn with its memory set to `0` while the `--workload-memory-maximum` is not set to `0` will roughly translate to "use as much memory as allowed". For example, say that we install Acorn with a non-zero `--workload-memory-maximum`.
+
+```shell
+acorn run --workload-memory-maximum 512Mi
+```
+
+Then we try to run an Acorn with its memory set to 0.
+
+```shell
+acorn run --memory 0 foo
+```
+
+When the `foo` Acorn gets provisioned, all of its containers will have their memory set to `512Mi` (the `--workload-memory-maximum` we set prior).
```

Comment:
if i omit  a memory setting completely in this case, will it get the max or be unset?

Reply:
In this case it will get the max.

Reply:
Misread the question - it will get whatever the default is here.